From 7da1cdcf67c8782d41d56c4b6fa8f967535e2be7 Mon Sep 17 00:00:00 2001
From: Rick Yiu <rickyiu@google.com>
Date: Fri, 16 Oct 2020 23:01:16 +0800
Subject: [PATCH 186/830] ANDROID: sched: Export symbols for vendor EAS wakup
 path function
X-NVConfidentiality: public

It needs to export some symbols so that vendor module could reference.

Bug: 170507310
Signed-off-by: Rick Yiu <rickyiu@google.com>
Change-Id: I1b3c3ea8d0a11c01a1ca9e124e93f85e52856dc4
(cherry picked from commit a61271a41a2c2825a51bd7655fc446e16d23f5f6)
Signed-off-by: Will McVicker <willmcvicker@google.com>
---
 kernel/sched/cpufreq_schedutil.c | 1 +
 kernel/sched/pelt.c              | 1 +
 2 files changed, 2 insertions(+)

diff --git a/kernel/sched/cpufreq_schedutil.c b/kernel/sched/cpufreq_schedutil.c
index 8038194f3bf5..83d41caafa2a 100644
--- a/kernel/sched/cpufreq_schedutil.c
+++ b/kernel/sched/cpufreq_schedutil.c
@@ -309,6 +309,7 @@ unsigned long schedutil_cpu_util(int cpu, unsigned long util_cfs,
 
 	return min(max, util);
 }
+EXPORT_SYMBOL_GPL(schedutil_cpu_util);
 
 static unsigned long sugov_get_util(struct sugov_cpu *sg_cpu)
 {
diff --git a/kernel/sched/pelt.c b/kernel/sched/pelt.c
index 2c613e1cff3a..c6cc21b9110b 100644
--- a/kernel/sched/pelt.c
+++ b/kernel/sched/pelt.c
@@ -306,6 +306,7 @@ int __update_load_avg_blocked_se(u64 now, struct sched_entity *se)
 
 	return 0;
 }
+EXPORT_SYMBOL_GPL(__update_load_avg_blocked_se);
 
 int __update_load_avg_se(u64 now, struct cfs_rq *cfs_rq, struct sched_entity *se)
 {
-- 
2.17.1

